code/modules/mob/living/basic/space_fauna/netherworld/migo.dm

```diff
@@ -41,26 +41,32 @@
 		new /mob/living/basic/migo/hatsune(get_turf(loc), mapload)
 		return INITIALIZE_HINT_QDEL
 
+	//if(istype(src, /mob/living/basic/migo/hatsune))
+		//var/static/list/death_loot = list(/obj/item/instrument/piano_synth)
+		//AddElement(/datum/element/death_drops, death_loot)
```

Comment:
Don't leave commented-out code in here if you aren't using it any more.

Reply:
Oops! forgot about that, nice catch

Reply:
You did not actually remove this